github_pr_to_internal_pr/github_pr_to_internal_pr.py

```diff
@@ -253,7 +237,8 @@ def main():
 
```

Comment:
Is it possible to set the MR title to `pr_title_desc + ' (Github PR)'` to make such MRs easier to distinguish? (one line above)